### PR TITLE
Bump GitHub package

### PR DESCRIPTION
**Pull request by mauricioszabo**

### CODE REVIEW

**Review by confused-Techie — APPROVED** [review]
This looks great to me and is an important update. Love seeing the ChangeLog being updated,

But otherwise do we have to keep the `0.36.12-fix`? Looks like a regular tag was also generated on [`github`](https://github.com/pulsar-edit/github/releases/tag/v0.36.12)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Pulsar next version
+- Bump github package to 0.36.12 (add Silicon Apple support)
```
Comment (confused-Techie) [inline CHANGELOG.md:2]:
```suggestion
- Bump github package to 0.36.13 (add Silicon Apple support)
```

File: package.json [hunk 2]
```diff
@@ -211,7 +211,7 @@
     "exception-reporting": "file:./packages/exception-reporting",
     "find-and-replace": "0.220.1",
     "fuzzy-finder": "1.14.3",
-    "github": "0.36.11",
+    "github": "0.36.12",
```
Comment (confused-Techie) [inline package.json:214]:
```suggestion
    "github": "0.36.13",
```

**Review by confused-Techie — CHANGES_REQUESTED** [review]
Hope that this new version improves things, which it seems that it is at least slightly. Just gotta keep the rest of it up to date with these new changes, then I say lets merge

**Comment by benonymus** [comment]
Is anything blocking this pr in particular?

**Comment by confused-Techie** [comment]
> Is anything blocking this pr in particular?

So this PR is essentially valid, but due to having to technically bump twice over the course of this PR, some of the other details are inaccurate as you can see in my review comments. 

Suppose we haven't gotten around to fixing those yet, but I may just commit my comments myself, and take a look at the tests how things are going.

I think the PR lost some steam since it didn't totally fix the issues we saw, but still wouldn't be bad to at least include another bump in here

**Comment by benonymus** [comment]
> > Is anything blocking this pr in particular?
> 
> So this PR is essentially valid, but due to having to technically bump twice over the course of this PR, some of the other details are inaccurate as you can see in my review comments.
> 
> Suppose we haven't gotten around to fixing those yet, but I may just commit my comments myself, and take a look at the tests how things are going.
> 
> I think the PR lost some steam since it didn't totally fix the issues we saw, but still wouldn't be bad to at least include another bump in here

What does it not fix? 

As far as I remember the only reason for the bump was to get to a version that supports apple silicon.
Which this version should do.
I have been using the apple silicon build form here: https://github.com/pulsar-edit/pulsar/pull/124 
for the last month every day without any issue.

**Comment by confused-Techie** [comment]
> 
> What does it not fix?
> 

Performance Issues mainly, but yeah fair enough it does tackle the main reason it was created. I'll go ahead and apply the reviews manually, and merge this one.

**Review by confused-Techie — APPROVED** [review]
The tests here look good, and like previously stated this adds support for Apple Silicon, I'll go ahead and merge

**Pull request merged**